packages/millicast-sdk/src/types/index.d.ts

```diff
@@ -125,7 +125,14 @@ declare module '@millicast/sdk' {
     /**
      * represents a collection of the webRTC stats collected before the diagnose call. 
      */
-    stats : Array<Object>
+    stats : Array<Object>,
+
+    /**
+     * Represents how long the stream has been connected for. If a stream disconnects and 
+     * re-connects a few times, this will reflect the duration only since the last time a 
+     * successful connection was established
+     */
+    connectionTime: Number
```

Comment:
I know this PR is already closed, but this attribute is not the same as the value returned, in Diagnostics method the attribute returned is `connectionDurationMs`

Reply:
I'll fix it in a follow up PR :) Thanks for picking this up @Santiago-Souto

Reply:
Here you go https://github.com/millicast/millicast-sdk/pull/412